Fixed navigation and added Erg-lang

jrrom opened the pull request:
Fixed #15 
Added Erg-lang #16

Commit:
Fixed navigation and added Erg-lang

Commit:
Fixed navigation and added Erg-lang

Commit:
Fixed navigation and added Erg-lang

Commit:
Fixed navigation and added Erg-lang

Commit:
Fixed navigation and added Erg-lang

vindarel commented:
Hi, thanks for the PR.

Erg has one difference compared to the languages of the first section, it is not implemented in Python. That distinction made sense to me at the time. How could we best show it (if that still makes sense)?

Maybe these sections:

1. Variants of Python made in Python. They can use Python libs.
2. ±= "Languages compatible with Python. They can use Python libs"

Or we just rely on your descriptions that clearly states it's built in Rust.

Also the detailed TOC is probably too much.

Commit:
Added new section and shortened Erg-lang description.

jrrom commented:
Thank you for your response. I have changed the following:

- Added new section "Implemented in another language but target the Python platform. They can use Python libs."
- I have shortened the language features.

What is a TOC? I am unaware of the full form.

vindarel commented:
Thanks!

(Table Of Content)
